src/main/java/teammates/common/datatransfer/attributes/EntityAttributes.java

```diff
@@ -57,4 +58,17 @@ public boolean isValid() {
      * e.g. trim strings
      */
     public abstract void sanitizeForSaving();
+
+    /**
+     * Return a copy of strings list, received in argument, with only non-empty string values remaining.
```

Comment:
Yes, that sounds simplier :) Just added "a list of" to that message: "Retrieves a list of nonempty errors from a list of error messages."